src/client/results/Results.js

```diff
@@ -1,128 +1,61 @@
-                isWordsToggledActionHide={mostFrequent.words.isToggledActionHide}
-                toggledUsers={mostFrequent.users.toToggle}
-                isUsersToggledActionHide={mostFrequent.users.isToggledActionHide}
-                paginationInfo={feed.paginationInfo}
-              />
-            </div>
-          </div>
-
-          <div className="hidden-md-down col-lg-3">
-            <div className="ui sticky">
-              <MostUsedWords filterTerm={mostFrequent.words.filterTerm}
-                wordInfoList={feed.groupedMostFrequentWords}
-                isWordsToggledActionHide={mostFrequent.words.isToggledActionHide}
-                postsLength={posts.length}
-                componentId="2"
-              />
-            </div>
-          </div>
-
-        </div>
-        <div className={`ui ${feed.posts.length > 0 && feed.fetchingRequestFromDB && 'active'} purple dimmer`}
-          style={{ position: 'fixed' }}
-        >
-          <div className="content">
-            <div className="center">
-              <h2 className="ui inverted icon header">
-                <div className="ui large text loader">Fetching posts from the database</div>
-              </h2>
-            </div>
+      <div className="row">
+        <div className="col-xs-12">
+          <div className="ui violet inverted center aligned segment">
+            <h3 className="ui inverted header">
+              Start using Socto by typing into the search bar or by using the filters.
+            </h3>
           </div>
         </div>
       </div>
     );
   }
-}
-Results.propTypes = {
-  feed: React.PropTypes.object,
-  mostFrequent: React.PropTypes.object,
+
+  return (
+    <div className="row">
+      <div className="ui fluid pointing two item top attached menu">
+        <a className={`item ${!showJournalismInfo && 'active'}`} onClick={onClickSocialWebResults}>
+          Social Web Results
+        </a>
+        <a className={`item ${showJournalismInfo && 'active'}`} onClick={onClickJournalismInfo}>
+          Journalism Information
+        </a>
+      </div>
+      <div className="ui bottom attached segment">
+        <div className={`ui tab ${!showJournalismInfo && 'active'}`} data-tab="social-web">
+          <SocialWebResults feed={feed} mostFrequent={mostFrequent}
+            socialWebVisible={!showJournalismInfo}
+          />
+        </div>
+        <div className={`ui tab ${showJournalismInfo && 'active'}`} data-tab="journalism-info">
+          Here will go the journalist info
+        </div>
+      </div>
+    </div>
+  );
 };
 
 const mapStateToProps = (state) => ({
+  searchTerms: state.searchTerms,
   feed: state.feed,
   mostFrequent: state.mostFrequent,
+  showJournalismInfo: state.showJournalismInfo,
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  onClickSocialWebResults: () => dispatch(changeResultsView(false)),
+  onClickJournalismInfo: () => dispatch(changeResultsView(true)),
```

Comment:
yer was thinking that, then was like, well there's never gonna be more than two tabs so seems a waste X)